perf/istio-install/setup_prometheus.sh

```diff
@@ -35,7 +35,7 @@ function install_prometheus() {
     kubectl wait --for=condition=Established -f https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/prometheusrule.crd.yaml
     kubectl wait --for=condition=Established -f https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/servicemonitor.crd.yaml
 
-    kubectl apply --namespace istio-prometheus -f "${PROM_OP}"
+    kubectl apply --namespace kube-system -f "${PROM_OP}"
```

Comment:
this will install everything in kube-system, which is not what we want. I think I was wrong when I was we could just remove this though, as it means it will be installed in whatever namespace we are currently in, rather than istio-prometheus.

We may just need to switch to using tiller? since it seems the operator was not made with `helm template` in mind at all